askama_shared/src/generator.rs

```diff
@@ -414,6 +417,38 @@ impl<'a> Generator<'a> {
         self.writeln("}");
     }
 
+    fn write_match(&mut self, state: &'a State, ws1: &WS, expr: &Expr, arms: &'a [When], ws2: &WS) {
+        self.handle_ws(ws1);
+        self.locals.push();
+        self.write("match ");
+        self.visit_expr(expr);
+        self.writeln(" {");
+
+        for arm in arms {
+            let &(ref ws, ref variant, ref params, ref body) = arm;
+            self.locals.push();
+            self.write(variant);
+            if params.len() > 0 {
+                self.write("(");
+                for param in params {
+                    self.locals.insert(param);
+                    self.write(param);
+                    self.write(", ");
```

Comment:
For readability of the generated code, I'd like to only emit the comma where necessary. I think the easiest way to do that is to `enumerate` over the loop iterator and emit a prefix comma for index > 0.